tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature

```diff
@@ -52,11 +52,11 @@ Feature: restrict Sharing
     And user "Carol" accepts the share "Shares/simple-folder" offered by user "Brian" using the sharing API in the server
     Then as "Carol" folder "/Shares/simple-folder" should exist in the server
 
-  @smokeTest
+  @smokeTest @skipOnOCIS
   Scenario: Forbid sharing with groups
     Given the setting "shareapi_allow_group_sharing" of app "core" has been set to "no" in the server
+#   allow shares with groups members is not available now in the oCIS webUI
```

Comment:
same here

Reply:
About this comment # shares only with groups members is not available now in the oCIS webUI i was just following the existing pattern # Reshare permission is not available now in the oCIS webUI